plugins/rapid7_insightidr/help.md

```diff
@@ -40,6 +47,411 @@ Example input:
+|metadata|investigation_metadata|True|The pagination parameters used to generate this page result|
+
+Example output:
+
+```
+{
+  "alerts": [
+    {
+      "alert_type": "Example Type",
+      "alert_type_description": "Example Description",
+      "created_time": "01-01-2020T00:00:00",
+      "detection_rule_rrn": {
+        "rule_name": "Example Rule Name",
+        "rule_rrn": {
+          "organizationId": "11111111-1111-1111-1111-111111111111",
+          "regionCode": "11-101",
+          "resource": "Example Resource",
+          "resourceTypes": [
+            "Example Type"
+          ],
+          "service": "Example Service"
+        }
+      },
+      "first_event_time": "01-01-2020T00:00:00",
+      "id": "11111111-1111-1111-1111-111111111111",
+      "latest_event_time": "01-01-2020T00:00:00",
+      "title": "Example Title"
+    }
+  ],
+  "metadata": {
+    "index": 0,
+    "size": 1,
+    "total_data": 1,
+    "total_pages": 1
+  }
+}
+```
+
+#### Set Priority of Investigation
+
+This action is used to change the priority of the investigation with the given ID or RRN.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|id|string|None|True|The ID or RNN of the investigation to change the priority of|None|rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111|
+|priority|string|None|True|Investigation's priority|['UNSPECIFIED', 'LOW', 'MEDIUM', 'HIGH', 'CRITICAL']|LOW|
+
+Example input:
+
+```
+{
+  "id": "rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111",
+  "priority": "LOW"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|investigation|investigation|True|The investigation for which the priority was set|
+
+Example output:
+
+```
+{
+  "investigation": {
+    "assignee": {
+      "email": "user@example.com",
+      "name": "Ellen Example"
+    },
+    "created_time": "2018-06-06T16:56:42Z",
+    "disposition": "BENIGN",
+    "first_alert_time": "2018-06-06T16:56:42Z",
+    "last_accessed": "2018-06-06T16:56:42Z",
+    "latest_alert_time": "2018-06-06T16:56:42Z",
+    "organization_id": "174e4f99-2ac7-4481-9301-4d24c34baf06",
+    "priority": "CRITICAL",
+    "rrn": "rrn:example",
+    "source": "ALERT",
+    "status": "OPEN",
+    "title": "Example Title"
+  }
+}
+```
+
+#### Set Disposition of Investigation
+
+This action is used to change the disposition of the investigation with the given ID or RRN.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|disposition|string|None|True|Investigation's disposition|['BENIGN', 'MALICIOUS', 'NOT_APPLICABLE']|BENIGN|
+|id|string|None|True|The ID or RNN of the investigation to change the disposition of|None|rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111|
```

Comment:
What is the default value? Since it is a required field can you please set the default value to something other than None